### PR TITLE
Explain R packages instead of database connections

### PR DESCRIPTION
**Pull request by akhagan**
While it is no doubt a useful feature of dplyr to "work directly with data stored in an external database", I think the discussion of this feature is more confusing than helpful at this point in the lesson (especially since it is never returned to or practiced later on). My suggestion (and what I've done here) is to spend a few moments discussing the concept of an R package using dplyr and tidyr as examples. This would arm beginning learners with the knowledge that R packages probably exist to support whatever their needs are in R so that they can seek them out as needed. 

I think it would be helpful to see the relational database portion expanded into another episode that walks the learner through the process of accessing and analyzing data from an external database using dplyr.

Also, I have changed the data frame referenced in line 96 (now line 92) from "surveys" to "interviews", the named data frame in the lesson.

### CODE REVIEW

**Comment by juanfung** [comment]
These are great suggestions but since they are not small changes to the substance of the discussion in episode 03, I'll have someone else take a look before merging: @francois or @ErinBecker ?

File: _episodes_rmd/03-dplyr-tidyr.Rmd [hunk 1]
```diff
@@ -39,33 +39,27 @@ source("../bin/download_data.R")
...
 
 The package **`tidyr`** addresses the common problem of wanting to reshape your data for plotting and use by different R functions. Sometimes we want data sets where we have one row per measurement. Sometimes we want a data frame where each measurement type has its own column, and rows are instead more aggregated groups. Moving back and forth between these formats is nontrivial, and **`tidyr`** gives you tools for this and more sophisticated  data manipulation.
 
+But there are also packages available for a wide range of tasks including building plots (**`ggplot`**, which we'll see later), downloading data from the NCBI database, or performing statistical analysis on your data set. Many packages such as these are housed on, and downloadable from, the **C**omprehensive **R** **A**rchive **N**etwork (CRAN) using `install.packages`. This function makes the package accessible by your R installation with the command `library()`, as you did with `tidyverse` earlier.
+
+To easily access the documentation for a package within R or R studio, use `help(package = "package_name")`.
```
Comment (ErinBecker) [inline _episodes_rmd/03-dplyr-tidyr.Rmd]: RStudio

File: _episodes_rmd/03-dplyr-tidyr.Rmd [hunk 2]
```diff
@@ -39,33 +39,27 @@ source("../bin/download_data.R")
...
 
-**`dplyr`** is a package for making tabular data manipulation easier by using a
-limited set of functions that can be combined to extract and summarize insights from your data. It pairs nicely with **`tidyr`** which enables you to
-swiftly convert between different data formats (long vs. wide) for plotting and analysis.
+**`dplyr`** is a package for making tabular data manipulation easier by using a limited set of functions that can be combined to extract and summarize insights from your data. It pairs nicely with **`tidyr`** which enables you to swiftly convert between different data formats (long vs. wide) for plotting and analysis.
 
 Similarly to **`readr`**, **`dplyr`** and **`tidyr`** are also part of the tidyverse. These packages were loaded in R's memory when we called `library(tidyverse)` earlier.
 
-## What are **`dplyr`** and **`tidyr`**?
+## What are R packages?
```
Comment (ErinBecker) [inline _episodes_rmd/03-dplyr-tidyr.Rmd]: What is an R package?

**Comment by ErinBecker** [comment]
Thank you @akhagan for catching that we hadn't introduced R packages yet! These changes look good to me. I found just a couple of small typos. @juanfung, could you make those changes and then merge? Let me know if you'd prefer for me to make these changes. Thanks!

**Comment by akhagan** [comment]
Thanks @juanfung  & @ErinBecker, I'm glad these were helpful changes!

**Pull request merged**